Essentials/src/com/earth2me/essentials/Settings.java

```diff
@@ -943,8 +943,28 @@ public long getAutoAfk() {
     }
 
     @Override
-    public long getAutoAfkKick() {
-        return config.getLong("auto-afk-kick", -1);
+    public long getAutoAfkKick(final User user) {
+        long limit = -1;
```

Comment:
I'm curious where -1 comes from.

Reply:
A setting of -1 means the player is never kicked. This is the default setting in the starting config.yml file, and was also the default in the code prior to our changes allowing ranked afkkick settings.